sql/src/main/java/io/crate/analyze/CreateSnapshotStatementAnalyzer.java

```diff
@@ -110,6 +114,7 @@ public CreateSnapshotAnalyzedStatement visitCreateSnapshot(CreateSnapshot node,
                             String.format(Locale.ENGLISH, "Cannot create snapshot of tables in schema '%s'", tableInfo.ident().schema()));
                 }
                 DocTableInfo docTableInfo = (DocTableInfo)tableInfo;
+                Relations.blockedRaiseException(new DocTableRelation(docTableInfo), Operation.READ);
```

Comment:
yes thought about the same, actually didn't get why it was like that but wasn't sure...